corehq/apps/users/static/users/js/roles.js

```diff
@@ -77,6 +77,16 @@ hqDefine('users/js/roles',[
             unwrap: function (self) {
                 var data = ko.mapping.toJS(self);
 
+                if (data.name) {
+                    // some older browsers don't use the String.prototype.trim() function. Explicity declare it in this case:
```

Comment:
Curious, how old are we talking about here? We have other js that uses trim.

Reply:
I know for certain that IE8 doesn't work with trim(). It's such a niche case that it's debatable whether it's actually needed, but I figured there was no harm in including it.
An alternative could be to use jQuery's trim `$.trim()`, which works with IE as well.

Reply:
Looks like [IE8 doesn't have it](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trim#Browser_compatibility).  I don't think we bother supporting that, is that correct, @orangejenny?

Reply:
Yeah, that's correct. This is weird feedback to give, because it's excellent to be thinking about browser support, but it's fine to remove the check.